### PR TITLE
ORM tweaks for Performance work

### PR DESCRIPTION
**Pull request by sc0ttkclark**
https://central.tri.be/issues/120395

### CODE REVIEW

File: src/Tribe/Repository.php [hunk 1]
```diff
@@ -606,13 +606,15 @@ public function found() {
 		$query = $this->build_query();
 
 		// The request property will be set during the `get_posts` method and empty before it.
-		if ( ! empty( $query->request ) ) {
+		if ( ! empty( $query->request ) && ( 0 < $query->found_posts || ! $this->skip_found_rows ) ) {
```
Comment (mitogh) [inline src/Tribe/Repository.php]: Probably personal preference but I think using `! $query->found_posts` is more expressive than `0 < $query->found_posts` and provides the same result as `0` will evaluate to `false`

Comment (sc0ttkclark) [reply src/Tribe/Repository.php]: ✅

**Comment by sc0ttkclark** [comment]
QA will be in branch on this one until we verify it's all stable.

**Comment by sc0ttkclark** [comment]
Made into the official release/G19.07 branch

**Pull request closed without merging**